packages/dataviews/src/types.ts

```diff
@@ -596,6 +597,29 @@ export interface ActionButton< Item > extends ActionBase< Item > {
 
 export type Action< Item > = ActionModal< Item > | ActionButton< Item >;
 
+export interface EmptyViewProps {
+	/**
+	 * The message to display when the view is empty.
+	 */
+	heading: string;
+
+	/**
+	 * The message to display when the view is empty.
+	 */
+	description?: string;
+
+	/**
+	 * The illustration to display when the view is empty.
+	 * It can be a string (URL) or a React element.
+	 */
+	illustration?: string | ReactElement;
+
+	/**
+	 * The actions to display when the view is empty.
+	 */
+	actions?: ReactElement< ComponentProps< typeof Button > >;
```

Comment:
This locks the buttons down to only `<Button>`s. On the one hand it forces consistency. On the other hand it could be too restrictive if the user had a `<Button>` wrapper for example.

Perhaps it is best to start restrictive and then open up if necessary. Because we can't go the other way without breaking changes.